Shop_Unity_base/Assets/Scripts/Shop/Model/Customer.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace Model
+{
+    using System;
+    using System.Collections.Generic;
+    using Utility;
+
+    public class Customer
+    {
+        private Inventory inventory;
+        public string Name { get; private set; }
+        public int Gold { get; private set; }
```

Comment:
Slight inconsistent naming scheme between this and the ShipModel (similar variable names there use lower case).

Reply:
It's the accessibility level. These are public properties, the ShopModel's were private variables.